org.scala-ide.sdt.core.tests/src/org/scalaide/core/testsetup/SDTTestUtils.scala

```diff
@@ -39,10 +39,11 @@ import org.eclipse.jdt.launching.JavaRuntime
 import org.scalaide.core.compiler.IScalaPresentationCompiler
 import org.scalaide.core.internal.project.ScalaProject
 
-/** Utility functions for setting up test projects.
+/**
+ * Utility functions for setting up test projects.
  *
  */
-object SDTTestUtils extends HasLogger {
+trait SDTTestUtils extends HasLogger {
```

Comment:
Why the change to a trait? Can there be/Do you want to have multiple instance of this type?